src/index.html

```diff
@@ -6,21 +6,428 @@
     <title>EasyHouse</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
+<body class="body_container">
     <h1>Easyhouse</h1>
-    <header>
+    <header class="project_menu">
         <a class="logo" href="#"><img src="images/logo.png" class="logoImg" alt="Logo"></a>
         <button type="submit"><img src="images/menu.png" alt="Menu"></button>
-        <nav>
-            <ul>
-                <li><a href="#">Home</a></li>
-                <li><a href="buy">Buy<img src="images/arrow-down.png" alt="Arrow down"></a></li>
-                <li><a href="about">About Us</a></li>
-                <li><a href="news">News</a></li>
-                <li><a href="contact">Contact</a></li>
+        <nav class="navbar">
```

Comment:
I noticed that you introduced a navbar class for styling purposes. However, it appears that this class duplicates the styles already applied to the nav element. Consider removing the navbar class to eliminate redundancy in your styles.